media-video/gxine/gxine-0.5.909.ebuild

```diff
@@ -13,27 +13,32 @@ SLOT="0"
 KEYWORDS="amd64 ppc ppc64 x86"
```

Comment:
revbump and drop to unstable, this is too invasive

Reply:
Done